Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:lts-stretch
+ARG JAVA_LIB_VERSION=1.29.0
+ARG PROTOC_VERSION=3.11.4
+RUN npm install ts-protoc-gen grpc-tools grpc
```

Comment:
```suggestion
```

Lets use the local ones in `./node/package.json`?

Reply:
agreed with @bobaaaaa to instead remove these depds from package.json and leave the version mngmt in `build_docker.sh`